charts/invenio/values.yaml

```diff
@@ -124,6 +124,8 @@ web:
       cpu: 1000m
       memory: 1Gi
   annotations: []
+  service:
+    type: NodePort
```

Comment:
This would be a breaking change to the chart's default behavior. I think it would be better to let the default value be empty (`type: ""`). You have already defined `ClusterIP` as default using the `default` function in the template, so having another default value here is a bit confusing.